eng/exit-dev-containers.ps1

```diff
@@ -0,0 +1,42 @@
+# # SPDX-License-Identifier: Apache-2.0
+# # Licensed to the Ed-Fi Alliance under one or more agreements.
+# # The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# # See the LICENSE and NOTICES files in the project root for more information.
+
+#Requires -Version 7
+# $ErrorActionPreference = "Stop"
+
+$docker=docker
+
+if ($a -gt 2) {
+  # Running under WSL1, need to reference docker.exe instead of docker. Simple
+  # alias doesn't work, sadly. This variable helps in the context of this
+  # script, but need to use "wsl1" commands with npm.
+  $docker=docker.exe
+}
```

Comment:
This is unnecessary when using PowerShell instead of Bash: `docker` command works just fine. Also `$docker` appears to be unused. I recommend simplifying the code below to just use `docker` everywhere.

Granted, this might be a problem if you start `pwsh` _from within a Bash prompt_ -  that is, PowerShell running on Linux. But there's a good work around - just stay in Bash at that point 😆.